test/_oncluster/scenario_runtime_test.go

```diff
@@ -0,0 +1,69 @@
+//go:build oncluster || runtime
+// +build oncluster runtime
+
+package oncluster
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	common "knative.dev/kn-plugin-func/test/_common"
+	e2e "knative.dev/kn-plugin-func/test/_e2e"
+)
+
+// TestRuntime will invoke a language runtime test against (by default) to all runtimes.
+// The Environment Variable E2E_RUNTIMES can be used to select the languages/runtimes to be tested
+func TestRuntime(t *testing.T) {
+
+	var runtimeList = []string{}
+	runtimes, present := os.LookupEnv("E2E_RUNTIMES")
+
+	if present {
+		if runtimes != "" {
+			runtimeList = strings.Split(runtimes, " ")
+		}
+	} else {
+		runtimeList = []string{"node", "python", "quarkus", "springboot", "typescript"} // "go" and "rust" pending support
```

Comment:
The problem doing that is it would bring "go" and "rust" and on cluster builds for those does not work so far (we shall need support for additional buildpacks first). Also I may incorporate soon in another PR tests for s2i and for s2i it is just a few set of runtimes as well...